project/accounts/factory.py

```diff
@@ -0,0 +1,73 @@
+from turtle import color
+import factory
+from . import models
+from categories.factory import CategoryFactory
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = models.User
+
+
+class ProfileFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = models.Profile
+
+    profile = factory.RelatedFactory(UserFactory, factory_related_name="profile")
+    first_name = factory.Faker('first_name')
+    last_name = factory.Faker('last_name')
+    about_me = factory.Faker('sentence', nb_words=20)
+
+    @factory.post_generation
+    def categories(self, create, extracted, **kwargs):
+        if not create or not extracted:
+            # Simple build, or nothing to add, do nothing.
+            return
+
+        # Add the iterable of categores using bulk addition
+        self.categories.add(*extracted)
+
+    #taggablemanager()
```

Comment:
```suggestion
```

Reply:
it is meant to refer to the tags field and to indicate its the code for that field